### PR TITLE
feat(journeyMaps): implement bbox event emitter

### PR DESCRIPTION
**Pull request by besartmemeti**

### CODE REVIEW

File: src/journey-maps/angular/journey-maps.ts [hunk 1]
```diff
@@ -147,6 +147,10 @@ export class SbbJourneyMaps implements OnInit, AfterViewInit, OnDestroy, OnChang
    * This event is emitted whenever the map is ready.
    */
   @Output() mapReady: ReplaySubject<MaplibreMap> = new ReplaySubject<MaplibreMap>(1);
+  /**
+   * This event is emitted whenever the bounds of the map has changed. (Whenever the map has been moved)
```
Comment (sebisteiner) [inline src/journey-maps/angular/journey-maps.ts:151]:
```suggestion
   * This event is emitted whenever the bounds of the map have changed. (Whenever the map has been moved)
```

File: src/journey-maps/angular/journey-maps.ts [hunk 2]
```diff
@@ -799,9 +803,10 @@ export class SbbJourneyMaps implements OnInit, AfterViewInit, OnDestroy, OnChang
       .pipe(debounceTime(200), takeUntil(this._destroyed))
       .subscribe(() => this.zoomLevelsChange.emit(this._getZooomLevels()));
 
-    this._mapCenterChangeDebouncer
-      .pipe(debounceTime(200), takeUntil(this._destroyed))
-      .subscribe(() => this.mapCenterChange.emit(this._map.getCenter()));
+    this._mapMovementDebouncer.pipe(debounceTime(200), takeUntil(this._destroyed)).subscribe(() => {
+      this.mapCenterChange.emit(this._map.getCenter());
+      this.mapBoundsChange.emit(this._map.getBounds());
```
Comment (sebisteiner) [inline src/journey-maps/angular/journey-maps.ts:808]: `getBounds()` returns a [`LngLatBounds`](https://maplibre.org/maplibre-gl-js-docs/api/geography/#lnglatbounds) object. I would suggest to emit `getBounds().toArray()`.

It's easier for our clients (especially the ones using the web component) to handle a plain array.

Comment (sebisteiner) [reply src/journey-maps/angular/journey-maps.ts:808]: We should also refactor `mapCenterChange` accordingly. (But not now as it is a breaking change)

**Review by sebisteiner — CHANGES_REQUESTED** [review]

**Review by sebisteiner — APPROVED** [review]
LGTM 👍

**Review by kyubisation — APPROVED** [review]
LGTM

**Pull request merged**